### ISSUE TITLE
Failing at "Add the official OpenVPN APT key"

### ISSUE THREAD

**Issue by ddvzlnz** [jlund/streisand#866]
This was working a couple of days ago.  Maybe the key has changed.

### Expected behavior:
add the apt key.
### Actual Behavior:
Fail (on Linode and Digital Ocean)

TASK [openvpn : Add the official OpenVPN APT key] *************************************************************************************
[0;31mfailed: [192.XXX.244.71] (item=-----BEGIN PGP PUBLIC KEY BLOCK-----[0m
[0;31mVersion: GnuPG v1[0m
[0;31m[0m
[0;31mmQENBE45PsIBCAC2K2LRZPQIUmJlCDKcncfR6vok2wowDpGpHZffvEEoUj/DoocR[0m
[0;31mLLpPHR5RB1zMWIs2IjF8vOtXMCBguDgtEvQTh6p6DM3D1fTnYp3pPlQyyzAuC81v[0m
[0;31mCQo44h09R4Nh2e38oMRVztmAnacC4g5aiSEamrZ4PbWdAdPc4uZdCPOGmUDJw8+q[0m
[0;31maAYvL/8pM7YqEu05FqE+aNcG02K+mDhA2bqRLLKoLEFpeMSO6vV8BrE7Vw1Rs1PM[0m
[0;31mVLDJt9HdXmC6vP+WWqDuj7/qfRb2wwlSIp5+aFyRHOUNyFKnWZYIObeV3+Y6oG6h[0m
[0;31mgmBtU1673mHDqVy26TwfjpJeudMKHVCrKXVXABEBAAG0QVNhbXVsaSBTZXBww6Ru[0m
[0;31mZW4gKE9wZW5WUE4gVGVjaG5vbG9naWVzLCBJbmMpIDxzYW11bGlAb3BlbnZwbi5u[0m
[0;31mZXQ+iQE+BBMBAgAoAhsDBgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgAUCU+B6sAUJ[0m
[0;31mC0rWWQAKCRCObai04VjFaU2sB/9eB1ZhA7u0o1XsiJpKcoEmcFS9EmYt+RjeplJP[0m
[0;31m0ewYoGFOGbEupvBdhqg1tqBxFw046kAbbBxwxU6NCSSPsIRR7FMLzF7JbRmlJXJc[0m
[0;31mFrl2WZaAL9wwoGucpGhWITUaipCXOcAY74WYWIx01mx0RxK8uRIq8lhsC4LUzAmg[0m
[0;31myA5H+nbF7A7FLtl9EUpnpqhnOC44ANlTSDo0whcrUoGaTvaepJrecypzQupMKYvd[0m
[0;31mcnraVXjTIr11CEXD8TS56sm9A3LPZTWD7jytUeRCh0kdixFI7ax1tkqDYtELfWg8[0m
[0;31m0YRWmT30o/k5hMZKWF5uwddbebfPB3EVHUtxzrnsxHReEVZBuQENBE45PsIBCADo[0m
[0;31mPgY5PFp2+3228AQw1GpS7CBIBVIbK0iw+VVWAfLTqu3Y7IXJKf9y8H5ErJlMxLVz[0m
[0;31mF8SGw1Djdb2wkF2ZUA3uxO7mOShTVsouPlCcCzK7PqKn3nblMw8UPr6Kv+jRpftq[0m
[0;31mJaner0/3P7j2/0T2aTnYKBJKhRDkzEVc9A2cAMs8mBFga8AxAjWxLannzCzyGlBl[0m
[0;31mM2b1HPNCPjGC+gM0Xv7rSFnAZ2dp8nWdEr1ikSdJTrOvXuiqBVXhOUF85k15I80m[0m
[0;31m0fXjdwA5to9BIy56bf80eIRfLOJeBy3UXUp+O7QHb2obDX4Fpjn4kA/G7Z7vA4jI[0m
[0;31miTHfCKVbGV/8wiW0lAoXABEBAAGJASUEGAECAA8CGwwFAlPgeu4FCQtK1qcACgkQ[0m
[0;31mjm2otOFYxWljKAgAq1qO1M+qnnnqgA1fMAlONmWwNiVsp5aTkXXcxR5BIHUQtkDU[0m
[0;31mN/T+HJuR72V+XeQdOb0NBFIGlSr670UPQKDkYoaxjCMUnNh8GLuekoGFnKEANB5t[0m
[0;31mKTSWS4XTAl7BVFnW3qHrFxOpMaP3zYHDeHjTO3kBQUR1GIFRxccY0/03DH+7u9XD[0m
[0;31mcJy7nPrj8usNKeszUWFn8ougSxy0z4odYb4I1yrVaE5+M+eYXukSn5EfZxnubMYo[0m
[0;31m8c1u+4CqwrxCBMncW5FozDzShEIew7hL/N1RddFfVCnmXGekgEKhGxEmv8TPVZ2E[0m
[0;31mQlEP0wog77si8RAyaoOJ0jkUr4ng6oxS/7QSmQ==[0m
[0;31m=ZLai[0m
[0;31m-----END PGP PUBLIC KEY BLOCK-----) => {"failed": true, "id": "E158C569", "item": "-----BEGIN PGP PUBLIC KEY BLOCK-----\nVersion: GnuPG v1\n\nmQENBE45PsIBCAC2K2LRZPQIUmJlCDKcncfR6vok2wowDpGpHZffvEEoUj/DoocR\nLLpPHR5RB1zMWIs2IjF8vOtXMCBguDgtEvQTh6p6DM3D1fTnYp3pPlQyyzAuC81v\nCQo44h09R4Nh2e38oMRVztmAnacC4g5aiSEamrZ4PbWdAdPc4uZdCPOGmUDJw8+q\naAYvL/8pM7YqEu05FqE+aNcG02K+mDhA2bqRLLKoLEFpeMSO6vV8BrE7Vw1Rs1PM\nVLDJt9HdXmC6vP+WWqDuj7/qfRb2wwlSIp5+aFyRHOUNyFKnWZYIObeV3+Y6oG6h\ngmBtU1673mHDqVy26TwfjpJeudMKHVCrKXVXABEBAAG0QVNhbXVsaSBTZXBww6Ru\nZW4gKE9wZW5WUE4gVGVjaG5vbG9naWVzLCBJbmMpIDxzYW11bGlAb3BlbnZwbi5u\nZXQ+iQE+BBMBAgAoAhsDBgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgAUCU+B6sAUJ\nC0rWWQAKCRCObai04VjFaU2sB/9eB1ZhA7u0o1XsiJpKcoEmcFS9EmYt+RjeplJP\n0ewYoGFOGbEupvBdhqg1tqBxFw046kAbbBxwxU6NCSSPsIRR7FMLzF7JbRmlJXJc\nFrl2WZaAL9wwoGucpGhWITUaipCXOcAY74WYWIx01mx0RxK8uRIq8lhsC4LUzAmg\nyA5H+nbF7A7FLtl9EUpnpqhnOC44ANlTSDo0whcrUoGaTvaepJrecypzQupMKYvd\ncnraVXjTIr11CEXD8TS56sm9A3LPZTWD7jytUeRCh0kdixFI7ax1tkqDYtELfWg8\n0YRWmT30o/k5hMZKWF5uwddbebfPB3EVHUtxzrnsxHReEVZBuQENBE45PsIBCADo\nPgY5PFp2+3228AQw1GpS7CBIBVIbK0iw+VVWAfLTqu3Y7IXJKf9y8H5ErJlMxLVz\nF8SGw1Djdb2wkF2ZUA3uxO7mOShTVsouPlCcCzK7PqKn3nblMw8UPr6Kv+jRpftq\nJaner0/3P7j2/0T2aTnYKBJKhRDkzEVc9A2cAMs8mBFga8AxAjWxLannzCzyGlBl\nM2b1HPNCPjGC+gM0Xv7rSFnAZ2dp8nWdEr1ikSdJTrOvXuiqBVXhOUF85k15I80m\n0fXjdwA5to9BIy56bf80eIRfLOJeBy3UXUp+O7QHb2obDX4Fpjn4kA/G7Z7vA4jI\niTHfCKVbGV/8wiW0lAoXABEBAAGJASUEGAECAA8CGwwFAlPgeu4FCQtK1qcACgkQ\njm2otOFYxWljKAgAq1qO1M+qnnnqgA1fMAlONmWwNiVsp5aTkXXcxR5BIHUQtkDU\nN/T+HJuR72V+XeQdOb0NBFIGlSr670UPQKDkYoaxjCMUnNh8GLuekoGFnKEANB5t\nKTSWS4XTAl7BVFnW3qHrFxOpMaP3zYHDeHjTO3kBQUR1GIFRxccY0/03DH+7u9XD\ncJy7nPrj8usNKeszUWFn8ougSxy0z4odYb4I1yrVaE5+M+eYXukSn5EfZxnubMYo\n8c1u+4CqwrxCBMncW5FozDzShEIew7hL/N1RddFfVCnmXGekgEKhGxEmv8TPVZ2E\nQlEP0wog77si8RAyaoOJ0jkUr4ng6oxS/7QSmQ==\n=ZLai\n-----END PGP PUBLIC KEY BLOCK-----", "msg": "key does not seem to have been added"}[0m

RUNNING HANDLER [dnsmasq : Restart dnsmasq] *******************************************************************************************

PLAY RECAP ****************************************************************************************************************************
[0;31m192.xxx.244.71[0m             : [0;32mok=19  [0m [0;33mchanged=14  [0m unreachable=0    [0;31mfailed=1   [0m
[0;33mlocalhost[0m                  : [0;32mok=9   [0m [0;33mchanged=3   [0m unreachable=0    failed=0   

### Steps to Reproduce:
1. Run streisand command and it will fail at 

TASK [openvpn : Add the official OpenVPN APT key] *************************************************************************************

**Comment by cpu** [comment]
> Output from git rev-parse HEAD in your Streisand directory :

^ Can you please share the above?

**Comment by ddvzlnz** [comment]
I think I have fixed this, or maybe it fixed itself.  I looked at the contents of the openvpn_signing.key file in the streisand>playbooks>roles>openvpn>files directory and it was different from the value I got if I looked at what  wget -O - https://swupdate.openvpn.net/repos/repo-public.gpg  gives me.  I put the new key in and the build completed without errors.  The existing file is this:  

-----BEGIN PGP PUBLIC KEY BLOCK-----
Version: GnuPG v1

mQENBE45PsIBCAC2K2LRZPQIUmJlCDKcncfR6vok2wowDpGpHZffvEEoUj/DoocR
LLpPHR5RB1zMWIs2IjF8vOtXMCBguDgtEvQTh6p6DM3D1fTnYp3pPlQyyzAuC81v
CQo44h09R4Nh2e38oMRVztmAnacC4g5aiSEamrZ4PbWdAdPc4uZdCPOGmUDJw8+q
aAYvL/8pM7YqEu05FqE+aNcG02K+mDhA2bqRLLKoLEFpeMSO6vV8BrE7Vw1Rs1PM
VLDJt9HdXmC6vP+WWqDuj7/qfRb2wwlSIp5+aFyRHOUNyFKnWZYIObeV3+Y6oG6h
gmBtU1673mHDqVy26TwfjpJeudMKHVCrKXVXABEBAAG0QVNhbXVsaSBTZXBww6Ru
ZW4gKE9wZW5WUE4gVGVjaG5vbG9naWVzLCBJbmMpIDxzYW11bGlAb3BlbnZwbi5u
ZXQ+iQE+BBMBAgAoAhsDBgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgAUCU+B6sAUJ
C0rWWQAKCRCObai04VjFaU2sB/9eB1ZhA7u0o1XsiJpKcoEmcFS9EmYt+RjeplJP
0ewYoGFOGbEupvBdhqg1tqBxFw046kAbbBxwxU6NCSSPsIRR7FMLzF7JbRmlJXJc
Frl2WZaAL9wwoGucpGhWITUaipCXOcAY74WYWIx01mx0RxK8uRIq8lhsC4LUzAmg
yA5H+nbF7A7FLtl9EUpnpqhnOC44ANlTSDo0whcrUoGaTvaepJrecypzQupMKYvd
cnraVXjTIr11CEXD8TS56sm9A3LPZTWD7jytUeRCh0kdixFI7ax1tkqDYtELfWg8
0YRWmT30o/k5hMZKWF5uwddbebfPB3EVHUtxzrnsxHReEVZBuQENBE45PsIBCADo
PgY5PFp2+3228AQw1GpS7CBIBVIbK0iw+VVWAfLTqu3Y7IXJKf9y8H5ErJlMxLVz
F8SGw1Djdb2wkF2ZUA3uxO7mOShTVsouPlCcCzK7PqKn3nblMw8UPr6Kv+jRpftq
Janer0/3P7j2/0T2aTnYKBJKhRDkzEVc9A2cAMs8mBFga8AxAjWxLannzCzyGlBl
M2b1HPNCPjGC+gM0Xv7rSFnAZ2dp8nWdEr1ikSdJTrOvXuiqBVXhOUF85k15I80m
0fXjdwA5to9BIy56bf80eIRfLOJeBy3UXUp+O7QHb2obDX4Fpjn4kA/G7Z7vA4jI
iTHfCKVbGV/8wiW0lAoXABEBAAGJASUEGAECAA8CGwwFAlPgeu4FCQtK1qcACgkQ
jm2otOFYxWljKAgAq1qO1M+qnnnqgA1fMAlONmWwNiVsp5aTkXXcxR5BIHUQtkDU
N/T+HJuR72V+XeQdOb0NBFIGlSr670UPQKDkYoaxjCMUnNh8GLuekoGFnKEANB5t
KTSWS4XTAl7BVFnW3qHrFxOpMaP3zYHDeHjTO3kBQUR1GIFRxccY0/03DH+7u9XD
cJy7nPrj8usNKeszUWFn8ougSxy0z4odYb4I1yrVaE5+M+eYXukSn5EfZxnubMYo
8c1u+4CqwrxCBMncW5FozDzShEIew7hL/N1RddFfVCnmXGekgEKhGxEmv8TPVZ2E
QlEP0wog77si8RAyaoOJ0jkUr4ng6oxS/7QSmQ==
=ZLai
-----END PGP PUBLIC KEY BLOCK-----

And the new file is this:

-----BEGIN PGP PUBLIC KEY BLOCK-----

mQENBE45PsIBCAC2K2LRZPQIUmJlCDKcncfR6vok2wowDpGpHZffvEEoUj/DoocR
LLpPHR5RB1zMWIs2IjF8vOtXMCBguDgtEvQTh6p6DM3D1fTnYp3pPlQyyzAuC81v
CQo44h09R4Nh2e38oMRVztmAnacC4g5aiSEamrZ4PbWdAdPc4uZdCPOGmUDJw8+q
aAYvL/8pM7YqEu05FqE+aNcG02K+mDhA2bqRLLKoLEFpeMSO6vV8BrE7Vw1Rs1PM
VLDJt9HdXmC6vP+WWqDuj7/qfRb2wwlSIp5+aFyRHOUNyFKnWZYIObeV3+Y6oG6h
gmBtU1673mHDqVy26TwfjpJeudMKHVCrKXVXABEBAAG0QVNhbXVsaSBTZXBww6Ru
ZW4gKE9wZW5WUE4gVGVjaG5vbG9naWVzLCBJbmMpIDxzYW11bGlAb3BlbnZwbi5u
ZXQ+iQFVBBMBAgA/AhsDBgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgBYhBDDr9Oc8
zmPu4STdJ45tqLThWMVpBQJZeJ2tBQkQ4vlrAAoJEI5tqLThWMVpPDUH/RdLsdG/
4kmal/rfbso3YVxZXGp2fHKrptvCVrUWluYs6H/XBV4x6aMe8Q6K7Qa7BSLA9jZ8
v+UN/4aA+urBcs6Ted/XbP3mKU47tOotW24nA1LRjd4gUSEXCaEOBbCSyw3uw6Vz
U1wr1gEmkC7kvBziL+Pcbt5tKTRhUfgbcjYNNdp/nAwn3Pm3OFRaBt/qDU2aYAOH
+k191x/ovDRO/UiU2CVvrdfv/VMZfo/rwxe8IiirxQ4k5DR2Vyu0DMNzlNTqRk8l
rUH0FBdl0rOiefH0m6ubKstpYCaOUYsh/FaW53O6qqrTlZqPtAav1cRog8zb8mhT
sFFAarhnZcQ/DG+5AQ0ETjk+wgEIAOg+Bjk8Wnb7fbbwBDDUalLsIEgFUhsrSLD5
VVYB8tOq7djshckp/3LwfkSsmUzEtXMXxIbDUON1vbCQXZlQDe7E7uY5KFNWyi4+
UJwLMrs+oqfeduUzDxQ+voq/6NGl+2olqd6vT/c/uPb/RPZpOdgoEkqFEOTMRVz0
DZwAyzyYEWBrwDECNbEtqefMLPIaUGUzZvUc80I+MYL6AzRe/utIWcBnZ2nydZ0S
vWKRJ0lOs69e6KoFVeE5QXzmTXkjzSbR9eN3ADm2j0EjLnpt/zR4hF8s4l4HLdRd
Sn47tAdvahsNfgWmOfiQD8btnu8DiMiJMd8IpVsZX/zCJbSUChcAEQEAAYkBPAQY
AQIAJgIbDBYhBDDr9Oc8zmPu4STdJ45tqLThWMVpBQJZeJ2DBQkQ4vlBAAoJEI5t
qLThWMVpCCIH+QFqEY+Xk5gJc10lbJUZEhJIknS/3GEd+3WBHgBtBaQCeK7+bFQP
ZagTN4SJLiwYcQDV04mZTpFOJV1k9AYaz7ENEjHe51mGhPM9sm5Ix7KwMNo0lHJ+
ryZ0zyie28IbGz+rYa7OdkhE2EmcQkezYNWC03G8yR9yGk3QZ3CtPPO/xYP2tBGc
OocqWUkVuR7KpitT9QnOZ4af26b83Vr/+qJ1FdSfW6/VAbyboVWya4oEnKSUusBm
0WCQzaLH15EpzgcdB/x8KVOTS1dAA5GNyRyhbRfP6yBXgBruCkPa4/np78/72jjW
vbAvOhOEMnfzWmf3VZq+q6hhIJf6Sp+dcoU=
=P3ax
-----END PGP PUBLIC KEY BLOCK-----
